src/media/js/addon/containers/__tests__/versionListing.test.js

```diff
@@ -59,8 +59,8 @@ describe('AddonVersionListing', () => {
                     showDeveloperActions={true}/>
     );
     const deleteDiv = ReactDOMHelper.queryClass(component,
-                                                'addon-version-delete');
-    const deleteBtn = ReactDOMHelper.queryTag(deleteDiv, 'button');
+                                                'version--header--buttons');
+    const deleteBtn = ReactDOMHelper.queryClass(deleteDiv, 'button--assertive');
```

Comment:
don't like that we're using a class name used for styling to identify a button, should be semantic